client/src/components/BookInfo.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components'
+
+export const Title = styled.h1`
+  margin-bottom: 0;
+  width: 100%;
+  font-weight: bold;
+  font-family: "Merriweather", "Georgia", serif;
```

Comment:
You could consider adding shared CSS stylings that are used multiple times into a parent component higher up. This way, you wouldn't have to repeatedly declare them in the children.
For example:
font-family is declared three times in Title, Author, Description. Might want to just have a parent styled component which contains Title, Author, Description